Clarify Developer to Starter upgrade does not include a free trial (#9430)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/billing-faqs.md

</Expandable>

> <Expandable alt_header="Do I get a free trial when upgrading from Developer to Starter?">

Reviewer: should also mention something here https://docs.getdbt.com/faqs/Accounts/platform-upgrade-instructions?version=2.0&name=Fusion?

> <Expandable alt_header="Do I get a free trial when upgrading from Developer to Starter?">

Reviewer: ```suggestion
<Expandable alt_header="What happens when I upgrade from Developer to Starter?">
```

Reviewer: sugg a more positive framing

> No. The free two-week Starter trial is for new accounts at signup, not for existing Developer plan customers upgrading to Starter.

Reviewer: ```suggestion
The free two-week trial is only available to new accounts at signup &mdash; but if you're already on the Developer plan, you can upgrade to Starter and immediately access all Starter features without needing to go through the trial.

```

> When you upgrade from Developer to Starter, you move directly to paid Starter billing. You'll need to enter your payment information and select your developer seats before your upgrade is complete. Your existing projects and account settings are not affected.

Reviewer: ```suggestion

When you upgrade from Developer to Starter, you move directly to paid billing &mdash; you'll enter your payment information and select your developer license seats to complete the upgrade. Your existing projects and settings carry over.
```
